valio-master/Valio/ScheduleTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ScheduleTableViewController.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/5/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+
+class ScheduleTableViewController: UITableViewController {
```

Comment:
In general I discourage using `UITableViewController` because the `self.view` in it is a `UITableView` not a `UIView` and this generates many problems on the way, if the screen grows. E.g. all the popup view, no connection screens, etc are subviewed to a subclass of `UIScrollView`, therefore they move when the user scrolls.

IMO it's better to do a `UIViewController` and just manually add the `UITableView` as a subview.